### PR TITLE
Fixed gem name

### PR DESCRIPTION
**Pull request by dinosaurjr10**
* Follow http://guides.rubygems.org/name-your-gem/

### CODE REVIEW

File: lib/env_hash_config/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module EnvHashConfig
-  VERSION = '1.1.0'
+  VERSION = '1.0.0'
```
Comment (johann-koebbe) [inline lib/env_hash_config/version.rb]: Is decrementing the version # intentional?

File: env_hash_config.gemspec [hunk 2]
```diff
@@ -9,7 +9,7 @@ Gem::Specification.new do |spec|
   spec.files = Dir['lib/**/*.rb']
   spec.homepage = 'https://github.com/dinosaurjr10/env_hash_config'
```
Comment (johann-koebbe) [inline env_hash_config.gemspec]: Homepage didn't change. You had it that way before? 😉

File: lib/env_hash_config/version.rb (hunk 1 above)
Comment (dinosaurjr10) [inline lib/env_hash_config/version.rb]: Yes. env_hash_config is brand new!

**Comment by johann-koebbe** [comment]
👍  Merge it!

**Pull request merged**